public/sass/_variables.generated.scss

```diff
@@ -112,7 +112,7 @@ $font-size-md: 14px !default;
 $font-size-sm: 12px !default;
 $font-size-xs: 10px !default;
 
-$line-height-base: 1.5 !default;
+$line-height-base: 1.5714285714285714 !default;
```

Comment:
I think we need to edit the template as per the instructions at the top of this file.

```
Edit grafana-ui/src/themes/_variables.scss.tmpl.ts to update template
```

then run `yarn themes:generate` to align template and generated file.

Reply:
@jackw not sure I understand what you mean, that command is run as part of yarn start. So it's already executed in this PR. I tried to execute it anyway but results in no changes

Reply:
@torkelo ignore me. I misunderstood the path these changes take to make their way here. 👍